src/store/reducers/pdisk/utils.ts

```diff
@@ -90,7 +90,7 @@ export function preparePDiskDataResponse([pdiskResponse = {}, nodeResponse]: [
 
     const diskSlots: PDiskData['SlotItems'] = [...vdisksSlots, ...emptySlots];
 
-    if (logSlot) {
+    if (logSlot && diskSlots.length > 0) {
```

Comment:
This condition is redundant, there should be no slots at all in case of `viewer/pdiskinfo`

Reply:
This function is used to prepare both old and new results.